scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
with gcc6, but not gcc. so gcc6 exists, but not gcc. Either way I was mistaken with my comment above, it actually built, but it build with cc, not the version of gcc I told it to build with.